docs-source/docs/modules/concepts/pages/pekko-management.adoc

```diff
@@ -0,0 +1,22 @@
+= Pekko Management
+
+
+include::partial$include.adoc[]
+
+{pekko-management}[Akka Management {tab-icon}, window="tab"] is a collection of tools to help operate Akka Cluster applications.
+
+== Management
+
+The main building block in {pekko-management}[Akka Management {tab-icon}, window="tab"] is the Management HTTP server.
```

Comment:
Should be 'Pekko Management'

Reply:
there are lots of other refs left behind to 'Akka'

Reply:
yeah, this PR is draft. When I am ready to review, I will remove the draft tag and call you to avoid wasting your time.

Besides a simple question, Do we need to keep the link to the Akka-Blog? I think we should keep it for now.

Reply:
it's ok to link to Akka docs when there is no Pekko equivalent - just make sure our docs still say it is about Akka